### DISCUSSION TITLE
BBS doors on Windows 10

### DISCUSSION THREAD

**Discussion by dustinnulf** [NuSkooler/enigma-bbs#378] [General]
Hi all, 

I just set up Enigma on a Windows10 machine. Looks cool! I can telnet into using "localhost"

Now I'd like to install a Door game; however, most documentation found is for Unix systems. Is there an example for Windows somewhere? I probably need DOSBox and a fossil driver... an example would be awesome.

Cheers,
Dustin

**Comment by NuSkooler** [comment]
Hi @dustinnulf ! Take a look here https://github.com/NuSkooler/enigma-bbs/blob/master/docs/modding/local-doors.md

For older DOS doors, you'll need a emulator that can bridge stdio to a COM port. DOSEMU does this, and I believe you can now do this with [DosBox-X](https://www.dosbox.com/)'s latest release. 

To run native Windows doors (e.g. 32bit doors with `DOOR32.SYS` support) you need something like `Bivrost!` also covered in the docs above.

**Reply by ellisgl** [reply]
How feasible would it to be to run something like LoRD 2 w/ multiplayer with this setup? I believe it uses SHARE.EXE to do file locking.

**Comment by NuSkooler** [comment]
@ellisgl Which setup? There are a number of ways. If you use DOSEMU for example, you get "SHARE" for free.

**Reply by ellisgl** [reply]
With dosbox-x

**Comment by NuSkooler** [comment]
@ellisgl Don't see why not. Map multiple serial -> COM1, COM2, ..., load up `SHARE.EXE`, and try it out 🎲
